### PR TITLE
New >>>>ALTERNATIVE<<<< roboticist jumpsuits

### PR DESCRIPTION
**Pull request by Wilchenx**
## About The Pull Request

Red and black is back baby

image

also replaces these and removes their crate (because noone ever spent 1500 credits on that shit)

image

## Why It's Good For The Game

People (including me) are nostalgic towards the old color scheme.

## Changelog
:cl:
add: Retro roboticist jumpsuits
del: mech suit crate
/:cl:

### CODE REVIEW

**Comment by zeskorion** [comment]
wilchen just wants peppermint back in the game

ignore he

**Comment by Wilchenx** [comment]
> wilchen just wants peppermint back in the game
> 
> ignore he

I comissioned the sprites before the jumpsuit was changed to purple, nice strawman I suppose.

**Comment by Naevii** [comment]
black and red are security colors and were changed for a reason to black and purple to fit science color theme. This color scheme is already used and well acknowledged by players and if used by other jobs its just confusing. 

Regarding the rest of the suits, They are cool, and looking good. But i dont see how they fit robotics. If its just for suit vendor, paid crate or beecoin shop then i dont see any issue.

**Comment by francinum** [comment]
Agreed. These fit better as meme items.

ngl their sprites are kinda hard to look at.

**Comment by Wilchenx** [comment]
> black and red are security colors and were changed for a reason to black and purple to fit science color theme. This color scheme is already used and well acknowledged by players and if used by other jobs its just confusing.

This is a alternative jumpsuit (read: it doesn't replace the current one) that you can buy from the vendor (just like old winter coats with old styles). I genuinely do not believe that anyone will be fooled to think that someone wearing this jumpsuit is a security officer.  Especially considering security officers wear other items that make them extremely obvious and they stand out as sore thumb with helmets, armor vests, jackboots, security belts, and red berets while roboticists wear labcoats covering most of the suit.


> Regarding the rest of the suits, They are cool, and looking good. But i dont see how they fit robotics. If its just for suit vendor, paid crate or beecoin shop then i dont see any issue.

They're a evangelion refrence (pilots riding giant robots), and yeah they're all in the robodrobe vendor as optional choice for the player.

**Comment by Naevii** [comment]
> 
> 
> > black and red are security colors and were changed for a reason to black and purple to fit science color theme. This color scheme is already used and well acknowledged by players and if used by other jobs its just confusing.
> 
> This is a alternative jumpsuit (read: it doesn't replace the current one) that you can buy from the vendor (just like old winter coats with old styles). I genuinely do not believe that anyone will be fooled to think that someone wearing this jumpsuit is a security officer. Especially considering security officers wear other items that make them extremely obvious and they stand out as sore thumb with helmets, armor vests, jackboots, security belts, and red berets while roboticists wear labcoats covering most of the suit.

well, they are distinctive because they follow the color scheme. black and red for security. in game where huge role is played by color schemes you cannot so easily disregard it. and for new player it sincerely might be confusing. It was for me when i started playing. 
> 
> > Regarding the rest of the suits, They are cool, and looking good. But i dont see how they fit robotics. If its just for suit vendor, paid crate or beecoin shop then i dont see any issue.
> 
> They're a evangelion refrence (pilots riding giant robots), and yeah they're all in the robodrobe vendor as optional choice for the player.

still fits better as beecoins shop thing than regular thing in vendor.

**Comment by BrumalLake** [comment]
i think this red and black being sec argument is ridiculous but
😂 runed metal golem
image

**Comment by PerishedFraud** [comment]
I don't like these. They look like teletubby suits. Nothing against robo getting more fashion but these are kinda icky. Get a resprite done please.

**Comment by utilbird** [comment]
get in the mech shinji

**Comment by PowerfulBacon** [comment]
This is quite possibly the ugliest sprite I have ever seen :key:
Also the >>>>ALTERNATIVE<<<< in the title makes it look like you know people aren't going to want it for some reason and want to have 'choice' as an excuse to implement something bad (since the only time anyone ever uses choice as an excuse for something is if its something nobody else wants).

image

**Comment by Wilchenx** [comment]
> This is quite possibly the ugliest sprite I have ever seen 🔑 Also the >>>>ALTERNATIVE<<<< in the title makes it look like you know people aren't going to want it for some reason and want to have 'choice' as an excuse to implement something bad (since the only time anyone ever uses choice as an excuse for something is if its something nobody else wants).
> 
> image

People kept thinking i'm replacing the current suits for some reason that's why I added a big disclaimer in the title

and yeah the sprites in the screenshot you posted are horrendous and that's why i'm replacing them.

**Comment by Fikou** [comment]
i think you forgot to replace the old icon files with new ones for the eva suits

**Comment by Wilchenx** [comment]
> i think you forgot to replace the old icon files with new ones for the eva suits

are you sure?

I just checked and it seems they've been replaced.

**Comment by Fikou** [comment]
> > i think you forgot to replace the old icon files with new ones for the eva suits
> 
> are you sure?
> 
> I just checked and it seems they've been replaced.

image
says created, not changed

**Comment by Wilchenx** [comment]
> > > i think you forgot to replace the old icon files with new ones for the eva suits
> > 
> > 
> > are you sure?
> > I just checked and it seems they've been replaced.
> 
> image says created, not changed

yeah some complete fool removed the sprites and noone ever noticed some item was missing them lol

**Comment by zeskorion** [comment]
> > > > i think you forgot to replace the old icon files with new ones for the eva suits
> > > 
> > > 
> > > are you sure?
> > > I just checked and it seems they've been replaced.
> > 
> > 
> > image says created, not changed
> 
> yeah some complete fool removed the sprites and noone ever noticed some item was missing them lol

Maybe cuz they sucked?

**Comment by Wilchenx** [comment]
> > > > > i think you forgot to replace the old icon files with new ones for the eva suits
> > > > 
> > > > 
> > > > are you sure?
> > > > I just checked and it seems they've been replaced.
> > > 
> > > 
> > > image says created, not changed
> > 
> > 
> > yeah some complete fool removed the sprites and noone ever noticed some item was missing them lol
> 
> Maybe cuz they sucked?

That's true but they forgot to remove the crate and associated items. (clueless)

**Review by PowerfulBacon — DISMISSED** [review]

**Review by ike709 — DISMISSED** [review]

File: code/modules/clothing/under/jobs/rnd.dm [hunk 1]
```diff
@@ -102,3 +102,14 @@
...
+/obj/item/clothing/under/rank/rnd/roboticist/retro
+	desc = "It's a slimming black jumpsuit with reinforced seams; great for industrial work. Vintage design, modern look, and the gloves are just for show."
+	name = "roboticist's retro jumpsuit"
+	icon_state = "robotics_retro"
+	item_state = "robotics"
+	item_color = "robotics_retro"
+	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+	fitted = NO_FEMALE_UNIFORM
+	alternate_worn_layer = GLOVES_LAYER
+	can_adjust = FALSE
```
Comment (Fikou) [inline code/modules/clothing/under/jobs/rnd.dm]:
```suggestion
	can_adjust = FALSE

```
needs a newline here to pass linters

**Comment by Archanial** [comment]
Doesn't pass linters

**Comment by Wilchenx** [comment]
I'll get on that today

**Pull request merged**